src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -287,16 +290,20 @@ def run_app_batch(
             raise transform_job_exception(e)
 
         new_job = Job(
-            job_id,
-            batch_app_id,
-            batch_params,
-            system_variable("user_id"),
-            tag=batch_method_tag,
-            app_version=batch_method_ver,
+            app_id=BATCH_APP["APP_ID"],
+            app_version=BATCH_APP["VERSION"],
             cell_id=cell_id,
+            extra_data={
+                # this data is not preserved in the ee2 record
+                "batch_app": app_id,
+                "batch_tag": tag,
+                "batch_size": len(params),
+            },
```

Comment:
This data is only available when the app is run. It doesn't get saved in ee2, so if the narrative is reloaded, all this handy info will be lost.